lib/global_uid/migration_extension.rb

```diff
@@ -3,7 +3,7 @@ module GlobalUid
   module MigrationExtension
 
     def create_table(name, options = {}, &blk)
-      uid_enabled = !(GlobalUid.configuration.disabled? || options[:use_global_uid] == false)
+      uid_enabled = !(GlobalUid.disabled? || options[:use_global_uid] == false)
```

Comment:
This paves the way for more readable code:
```suggestion
      uid_enabled = GlobalUid.enabled? && options[:use_global_uid] != false
```
Why do we compare the `options` to a boolean? Is it a guard against `nil`? Or could we just do this instead?

```suggestion
      uid_enabled = GlobalUid.enabled? && options[:use_global_uid]
```

Reply:
The value of `options[:use_global_uid]` is either `nil` or `false`. If you're using the `global_uid` gem and have it enabled, all migrations by default will use this extension unless you specifically disable it for that table.

Reply:
Which I found confusing, since the `drop_table` checks for it to be `true`
https://github.com/zendesk/global_uid/blob/6705d48af1f4accf5d112b9f43c8889c5abc5753/lib/global_uid/migration_extension.rb#L32-L33